nestkernel/node.h

```diff
@@ -830,6 +830,13 @@ class Node
    */
   virtual index get_local_device_id() const;
 
+  /**
+   * Difference between threshold and membrane potential for given time step.
+   * @param   double time step
+   * @returns difference between updated membrane potential and threshold
+   */
+  virtual double threshold_distance( double t_step ) const;
```

Comment:
Actually, I wonder if we need `threshold_distance` as a virtual function in `Node` at all: Since `regula_falsi` has the node class as a template argument, it should work with all node classes providing `threshold_distance`, and for all others, it does not matter.

Reply:
When doing the pointer to member this seemed necessary, but I will try now with just pointing to the node!